### ISSUE TITLE
Render docs for experimental namespace

### ISSUE THREAD

**Issue by oruebel** [hdmf-dev/hdmf-common-schema#56]
The docs for the new experimental namespace are currently not being rendered on ReadTheDocs. This will likely require some changes to the setup of how the docs are generated and/or the hdmf-docutils.

**Comment by mavaylon1** [comment]
@oruebel Is this still needed?

**Comment by oruebel** [comment]
Yes, this is still an issue. The problem is that the docs only process the main namespace `hdfm-common` 

https://github.com/hdmf-dev/hdmf-common-schema/blob/b3e48fcc5fff10dce0585d57b84cfed5816089a3/common/namespace.yaml#L3

while the second namespace `hdfm-experimental` is being ignored:

https://github.com/hdmf-dev/hdmf-common-schema/blob/b3e48fcc5fff10dce0585d57b84cfed5816089a3/common/namespace.yaml#L28

As a result, `ExternalResources` and `EnumData` are not being rendered on the page, e.g., as part of the type hierarchy:

https://hdmf-common-schema.readthedocs.io/en/latest/format.html#type-hierarchy

Fixing this for the docs, will require some trickery on the either the setup here in `conf.py` or potentially on the `hdfm-docutils` to support multiple namespaces.
